src/fides/api/util/tcf/tcf_experience_contents.py

```diff
@@ -564,7 +562,11 @@ def get_tcf_contents(
         matching_privacy_declaration_query=matching_privacy_declarations,
     )
 
-    populate_vendor_relationships_basic_attributes(
+    # Finally, add missing TCFVendorRelationships records for vendors that weren't already added
+    # via the special_features, features, and special_purposes section.  Every vendor in the overlay
+    # should show up in this section. Add the basic attributes to the vendor here to avoid duplication
+    # in other vendor sections.
```

Comment:
🙏 thank you @pattisdr for the extra care in adding this comment, will help make sense of things moving forward!